src/app/admin/pending-users-table/pending-users-table.component.html

```diff
@@ -17,95 +17,209 @@
+        </td>
+      </ng-container>
+      <ng-container matColumnDef="organisation">
+        <th
+          mat-header-cell
+          *matHeaderCellDef
+          mat-sort-header="organisation"
+          [disabled]="!hideFilters"
+          style="cursor: pointer"
+          columnName="organisation"
+          scope="col"
+        >
+          <span [hidden]="!hideFilters">Organisation</span>
+          <div [hidden]="hideFilters">
+            <mat-form-field class="filter" floatLabel="never">
+              <mat-label>Organisation</mat-label>
+              <input
+                #filters
+                matInput
+                name="organisation"
+                (keyup)="applyFilter()"
+              />
+            </mat-form-field>
+          </div>
+        </th>
+        <td
+          mat-cell
+          *matCellDef="let record"
+          style="text-align: left; word-wrap: break-word"
+        >
+          <span>{{ record.organisation }}</span>
+        </td>
+      </ng-container>
+      <ng-container matColumnDef="actions">
+        <td
+          mat-header-cell
+          *matHeaderCellDef
+          columnName="actions"
+          style="width: 7%"
+        ></td>
+        <td
+          mat-cell
+          *matCellDef="let record; let i = index"
+          style="text-align: center"
+        >
+          <button mat-icon-button color="primary" [matMenuTriggerFor]="actions">
+            <span class="fas fa-ellipsis-v"></span>
+            <span style="display: none">Menu</span>
+          </button>
+          <mat-menu #actions="matMenu" yPosition="below" xPosition="after">
+            <button
+              mat-menu-item
+              color="accent"
+              type="button"
+              (click)="askForSoftApproval(record)"
+            >
+              <span class="fas fa-check"></span> Approve user
+            </button>
+            <button
+              mat-menu-item
+              color="warn"
+              type="button"
+              (click)="askForSoftRejection(record)"
+            >
+              <span class="fa fa-ban warning"></span> Reject user
+            </button>
+          </mat-menu>
+        </td>
+      </ng-container>
+      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+      <tr mat-row *matRowDef="let record; columns: displayedColumns"></tr>
+    </table>
 
-        <div class="bordered is-hidden pxy-2" [ngClass]="{'block':!totalItemCount && !isLoadingResults}">
-            <p class="mt-2">There are no pending users in this repository.</p>
-        </div>
-        <div class="mdm--mat-pagination" [ngClass]="{'is-hidden':totalItemCount < 6}">
-            <mdm-paginator [length]="totalItemCount" showFirstLastButtons></mdm-paginator>
-        </div>
+    <div
+      class="bordered is-hidden pxy-2"
+      [ngClass]="{ block: !totalItemCount && !isLoadingResults }"
+    >
+      <p class="mt-2">There are no pending users in this model.</p>
```

Comment:
This sentence also doesn't fit the context. Perhaps change to "There are no pending users to display."